<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>831</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color:rgb(20, 18, 29)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>630</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
	border-radius:16px;
	background-color: rgb(24, 22, 29);
}
QPushButton::pressed
{
	background-color:rgb(95, 95, 95)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/Icona Indietro.png</normaloff>../img/Icona Indietro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../img/Icona Comunicazioni.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>621</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);
background-color:rgba(0, 192, 225, 125)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="mailField">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border:none;
border-bottom: 1px solid rgba(255, 255, 255, 100);
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>E-mail Biblioteca</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Destinatari/o</string>
    </property>
   </widget>
   <widget class="QComboBox" name="recipientBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="allBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Inoltra a tutti</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;
border-style: outset;
border-width: 2px;
border-color:rgb(27,25,39);
background-color:rgba(0, 192, 225, 125)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="textMail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>551</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(23, 28, 78, 100);
color:rgb(255, 255, 255);
border-radius: 20px;
border-style: solid;
border:none;
text-align: center;
border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border: 1px solid rgba(255, 255, 255, 100);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Testo mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Oggetto mail</string>
    </property>
   </widget>
   <widget class="QComboBox" name="messageBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
background-color:transparent;</string>
    </property>
    <property name="text">
     <string>Messaggi Predefiniti</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="objectMail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(23, 28, 78, 100);
color:rgb(255, 255, 255);
border-radius: 20px;
border-style: solid;
border:none;
text-align: center;
border:none;
background-color: rgba(23, 28, 78, 17);
color:rgb(255, 255, 255);
font: 14pt &quot;Source Code Pro for Powerline&quot;;
border: 1px solid rgba(255, 255, 255, 100);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshtextButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton 
{
background-color: #1A1927;
color:rgb(255, 255, 255);
border-radius:15px;
border: none

}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
    </property>
    <property name="text">
     <string>Aggiorna Messaggio pred.</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>650</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{
background-color: #1A1927;
color:rgb(255, 255, 255);
border-radius:15px;
border: none

}
QPushButton::pressed
{
	background-color:rgb(30, 97, 255)
}</string>
   </property>
   <property name="text">
    <string>Invia</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/send.png</normaloff>../img/send.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>921</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../img/Sfondo.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../img/recipient.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../img/oggetto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_7</zorder>
  <zorder>frame_3</zorder>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>sendButton</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
